### ISSUE TITLE
Tag positions changed using Anchor

### ISSUE THREAD

**Issue by jselner** [docusign/docusign-csharp-client#231]
Converting from 3.1.5 to 4.0.0 all the Anchor Tag placements were off. Seemed like all the Anchor Offsets were increased by a percent, maybe AnchorUnits is not working correctly?

Code Example:
SignHere signAnchor = new SignHere
            {
                AnchorString = anchorMapping.Trim(),
                AnchorXOffset = offsetX.ToString(),
                AnchorYOffset = offsetY.ToString(),
                AnchorUnits = "pixels",
                AnchorIgnoreIfNotPresent = "true",
                DocumentId = docID.ToString(),
                RecipientId = recID.ToString()
            };

**Comment by LarryKlugerDS** [comment]
I raised your question with our engineering group. Their reply:

>AnchorOffset Defaults are still 0.
> 
>I would guess they are seeing a difference in a tab saved in a template that has non default offsets (ie greater than zero).
>If this were off , I would expect the X/Y position to be off as well.
> 
>I assume they are moving from REST v2 to v2.1. 
>The description is really brief.  I don’t know if they are applying templates, creating templates vs creating envelopes, using templates or applying templates.
> 
>Generally, we trying to keep anchor tab version placement unchanged by honoring the AnchorTabVersion read_only property.

**Comment by jselner** [comment]
Hi Larry, let me clarify. Creating an envelope I place tags on the document using AnchorString then set the AnchorXOffset and AnchorYOffset to adjust the position based on that AnchorString. However upon upgrading the most recent version which uses REST v2.1 all the offsets were off. Seemed to be multiplied by a percent, so wondering if AnchorUnits is not being applied correctly or there's some other scaling issue.

As an example I use the following two tags to create a signature line. Matching on the signer's name and then moving the signer tag to the upper left and date signed tag to upper right. After upgrading the tags are no longer placed in the correct spots moving too far to the left and right and too far up.

SignHere signAnchor = new SignHere
{
AnchorString = "Roger  L Leatherwood",
AnchorXOffset = "-90",
AnchorYOffset = "-6",
AnchorUnits = "pixels",
AnchorIgnoreIfNotPresent = "true",
DocumentId = "1",
RecipientId = "1"
};

            DateSigned dateAnchor = new DateSigned
            {
                AnchorString = "Roger  L Leatherwood",
                AnchorXOffset = "170",
                AnchorYOffset = "-17",
                AnchorUnits = "pixels",
                AnchorIgnoreIfNotPresent = "true",
                Font = "Arial",
                FontSize = "Size9",
                DocumentId = "1",
                RecipientId = "1"
            };

How it should look:
![image](https://user-images.githubusercontent.com/3810102/62081107-95e02f00-b217-11e9-86d6-142595ecf320.png)

**Comment by LarryKlugerDS** [comment]
So you're saying that to get the same effect on the document, you've needed to change the offsets as your show in your two versions of the code?

Did you turn on responsive signing or make any other change in the envelope while upgrading to  v2.1? 
Thank you,
Larry

**Comment by jselner** [comment]
Yes all the offsets were off after upgrading from the 3.1.5 version of the sdk to 4.0.0 (v2.1 of api). And there was no real changes, only some models in 4.0.0 had different parameters i needed to account for.

**Comment by LarryKlugerDS** [comment]
Good news, DocuSign Engineering has confirmed the issue as a breaking change for updating to v2.1. I've filed TT-2616 to track the issue. I should have more news tomorrow.

Thank you again for the problem report.

**Comment by LarryKlugerDS** [comment]
Work is underway to exactly document the positioning offsets in the different situations.
